src/components/PostComment/PostComment.jsx

```diff
@@ -0,0 +1,91 @@
+import axios from 'axios';
+import { useState, useContext } from 'react';
+import './PostComment.css';
+import { DefaultUserContext } from '../../Context/DefaultUserContext';
+
+const PostComment = ({
+	review_id,
+	allComments,
+	setAllComments,
+}) => {
+	const [error, setError] = useState();
+	const [isSubmitted, setIsSubmitted] = useState(false);
+	const [body, setBody] = useState('');
+	const { user } = useContext(DefaultUserContext);
+
+	const SubmitComment = (event) => {
+		event.preventDefault();
+		if (body.length < 50) {
+			setError(
+				'Your comment needs top at least 50 characters'
+			);
```

Comment:
this message only appears very briefly - would be nice to keep it visible a bit longer!

Also it would be a good idea to inform the user of what they need to do *before* they are able to make a mistake